astropy/nddata/blocks.py

```diff
@@ -113,12 +113,12 @@ def block_reduce(data, block_size, func=np.sum):
         ``block_size`` will be used for for every axis.
 
     func : callable, optional
-        The method to use to downsample the data.  Must be a callable
-        that takes in a `~numpy.ndarray` along with an ``axis`` keyword,
-        which defines the axis or axes along which the function is
-        applied.  The ``axis`` keyword must accept multiple axes as a
-        tuple.  The default is `~numpy.sum`, which provides block
-        summation (and conserves the data sum).
+        The method to use to downsample the data. Must be a callable
+        that takes in a 4D `~numpy.ndarray` and has an ``axis``
```

Comment:
Must it always be 4D? I thought I can pass in 2D too.

Reply:
Yes.  The `block_reduce` function takes a 2D array, but reshapes it as 4D (using [`reshape_as_blocks`](https://docs.astropy.org/en/stable/api/astropy.nddata.reshape_as_blocks.html?highlight=reshape_as_blocks)) and then passes the 4D array to the `func` callable with `axis=(2,3)` to get back a reduced 2D array.  Make sense?

Reply:
OK, I updated the docstring.